src/Features/Core/Portable/FindUsages/DefinitionItem.DefaultDefinitionItem.cs

```diff
@@ -24,14 +24,14 @@ internal sealed class DefaultDefinitionItem(
             ImmutableArray<string> tags,
             ImmutableArray<TaggedText> displayParts,
             ImmutableArray<TaggedText> nameDisplayParts,
-            ImmutableArray<TaggedText> originationParts,
             ImmutableArray<DocumentSpan> sourceSpans,
             ImmutableArray<ClassifiedSpansAndHighlightSpan?> classifiedSpans,
+            ImmutableArray<AssemblyLocation> metadataLocations,
```

Comment:
nit: assemblylocations. (applies everywhere).  alternatively, rename AssemblyLocation to MetadataLocation.  Important bit is keeping the namign consistent.

Reply:
We already have MetadataLocation type.

Reply:
perhaps metadataAssemblyLocations then :)

Reply:
Yeah, that's too long... 
I think it's actually fine as is: `assemblyLocations` wouldn't convey the fact that the parameter is only for metadata assemblies, not source assemblies. `AssemblyLocation` type itself isn't necessarily limited to metadata assemblies. We may use it elsewhere for source assemblies if needed.

Reply:
wfm.